packages/backend/src/accounting/psql/ledger-transfer/index.test.ts

```diff
@@ -59,9 +395,9 @@ describe('Ledger Transfer', (): void => {
       async ({ accountType }): Promise<void> => {
         const transfer = await createLedgerTransfer(
           {
-            ledger: creditAccount.ledger,
-            creditAccountId: creditAccount.id,
-            debitAccountId: debitAccount.id,
+            ledger: account.ledger,
+            creditAccountId: account.id,
+            debitAccountId: peerAccount.id,
```

Comment:
I think `creditAccount`/`debitAccount` might be clearer.
What if you defined them in a `beforeEach` from `account`/`peerAccount`?